site/src/examples/dzi/dzi-demo.tsx

```diff
@@ -90,7 +91,7 @@ export function DziDemo() {
             <p>Scroll below to view image</p>
             <div style={{ display: 'flex', flexDirection: 'row' }}>
                 {images.map((v) => (
-                    <div key={v.imagesUrl} style={{ width: screenSize[0], height: screenSize[1] }}>
+                    <div key={v.imagesUrl} style={{ width: screenSize[0], height: screenSize[1], marginTop: 0 }}>
```

Comment:
Needed to override some Starlight CSS that was making the two DZI viewers not line up with each other